plugins/accelerated-peft/pyproject.toml

```diff
@@ -26,12 +26,13 @@ classifiers=[
 
 [project.optional-dependencies]
 flash-attn = ["flash-attn"]
+auto_gptq = ["auto_gptq @ git+https://github.com/AutoGPTQ/AutoGPTQ.git"]
```

Comment:
you need to have the hash here

Reply:
I havent tested it but the main branch seemed to have pushed a fix since